queue/remote_base.go

```diff
@@ -494,7 +506,10 @@ func (q *remoteBase) monitorStaleRetryingJobs(ctx context.Context) {
 		}
 	}()
 
-	const monitorInterval = time.Second
+	monitorInterval := time.Second
```

Comment:
Can we specify somewhere in the documentation that the default is 1 sec?

Reply:
I made it a constant and added some docs.